framework/TApplication.php

```diff
@@ -979,6 +979,7 @@ public function applyConfiguration($config, $withinService = false)
 				foreach ($parameter[1] as $name => $value) {
 					$component->setSubProperty($name, $value);
 				}
+				$component->dyInit();
```

Comment:
This is ok by now; For the future, it would be nice to investigate a more generic way to call these dynamic methods.

Reply:
I'll check in the TTheme behavior (when this is resolved) enabling with dynamic events through it for how dynamic events should be used.  dynamic events have multiple use cases. I believe this is one of them.  I'm open to rethinking these things.  I'm not sure I understand making the dynamic event method space more generic.

Reply:
I see this as a valid use case for intra-object communication via 'dy' events.  It may look like a method but it's an event.